### ISSUE TITLE
Not supported 1809

### ISSUE THREAD

**Issue by Rene-Moll** [SobieskiCodes/RDPWrap#1]
Hi, did everything as you described it in the readme. It still says "Not supported" sadly. Win vers. 1809. Hyper-V is turned on. Any ideas?

**Comment by SobieskiCodes** [comment]
> 
> 
> Hi, did everything as you described it in the readme. It still says "Not supported" sadly. Win vers. 1809. Hyper-V is turned on. Any ideas?

Home or pro and what OS build? eg; 18362.418 can be found under computer properties.
Also, 32 or 64 bit?
I don't have an 1809 machine but I can boot one up. This is specifically for 1903.

**Comment by Rene-Moll** [comment]
Hi SobieskiCodes, thanks for your reply. I could solve it on my own now. I am using Win 10 Pro 64bit. I copied the .ini file from the blog here: [](https://www.andysblog.de/windows-probleme-beim-rdp-wrapper-loesen)
Under the headline of Update 21.10.2019 (Für Windows 10 1903 (10.0.17763.771))

My machine says it is 1809, but RDP Wrapper Config says I am using 10.0.17763.771. No idea in what relation those 2 version numbers stand, but I tried my luck with that .ini and then it was working.
